src/renderer/map/Map.js

```diff
@@ -1,68 +1,42 @@
 import { ipcRenderer, remote } from 'electron';
 import fs from 'fs';
 import L from 'leaflet';
-import path from 'path';
 import React, { Component, createRef } from 'react';
 import { Map, Marker, Popup } from 'react-leaflet';
 
+import { cache, images, locations, startLocation } from '../../../resources/index.js';
 import CachedTileLayer from './CachedTileLayer.js';
 import GeolocationControl from './GeolocationControl.js';
 
-import 'leaflet/dist/leaflet.css';
 import './map.css';
 
-const vehicleIcons = {};
```

Comment:
why are these not field variables?

Reply:
Each vehicle will have a certain icon. For sake of example we will use UAV.

UAV that is connected will have a black border around it.
UAV that is disconnected with have a red border around it.

We can add others as we go but that's kinda where we are right now.

Reply:
We will also have the image to have the format of the vehicle platform.

We have uav.png for all UAVs. We plan to create a multirotor.png for all multirotor vehicles. These will have black borders.

All vehicles with red borders will have _red appended to them. For example: uav_red.png, multirotor_red.png

Software will automatically assign images to the vehicles hence.